Userland/Libraries/LibWeb/Painting/SVGGeometryPaintable.h

```diff
@@ -15,41 +15,6 @@ class SVGGeometryPaintable final : public SVGGraphicsPaintable {
     JS_CELL(SVGGeometryPaintable, SVGGraphicsPaintable);
 
 public:
-    class PathData {
```

Comment:
could you remove `PathData` from https://github.com/SerenityOS/serenity/pull/21681/commits/2911830adedf24a5d792213c70cfbf8601a91ef3 so it is not introduced in one commit and then immediately removed in the next one?